examples/full-page-cache/README.md

```diff
@@ -0,0 +1,32 @@
+# Hydrogen example: Full-page Cache
+
+This is an exmaple of using full page cache with Hydrogen. It is enabled for all pages other than the accounts and cart routes. It massively improves the performance of the rest of the app, but has the following implications:
```

Comment:
Should  add a note about "Only use full-page cache if you can cache most of your pages. Otherwise, the non-cached pages will be a bit slower than if you don't cache anything".

Reply:
"most" or you mean "full page"?
Why would full pages effect the rest of the non-cached pages?